jkube-kit/doc/src/main/asciidoc/inc/enricher/image/_jkube_image.adoc

```diff
@@ -0,0 +1,22 @@
+[[jkube-image]]
+==== jkube-image
+
+This enricher merges in container image related fields into specified controller (e.g `Deployment`, `ReplicaSet`, `ReplicationController` etc.) Pod specification.
+
+- The full image name is set as `image`.
+- An image alias is set as `name`. If alias isn't provided, then opinionated name using image user and project name is used.
+- The pull policy `imagePullPolicy` is set according to the given configuration. If no
```

Comment:
I am confused here:
-  `imagePullPolicy` or `.jkube-image.pullPolicy`?
- any reason why there is only pullpolicy in the configuration options ?

Reply:
I'm not sure. I ported most of the documentation from enricher javadoc : https://github.com/eclipse/jkube/blob/master/jkube-kit/enricher/generic/src/main/java/org/eclipse/jkube/enricher/generic/ImageEnricher.java#L59